postgres/tests/test_statements.py

```diff
@@ -793,6 +868,79 @@ def wait(conn):
         blocking_conn.close()
 
 
+@pytest.mark.parametrize("pg_stat_activity_view", ["pg_stat_activity", "datadog.pg_stat_activity()"])
+@pytest.mark.parametrize(
+    "set_reported_hostname,expected_reported_hostname",
+    [
+        (True, 'fred_hostname_override'),
+        (False, 'stubbed.hostname'),
+    ],
+)
+def test_activity_reported_hostname(
```

Comment:
Same comment as above. Can we come up with a simpler test that just produces a single activity event? We don't need all the same involved setup to test that we're capturing activity data correctly for this test.